saleor/plugins/webhook/plugin.py

```diff
@@ -1245,6 +1282,34 @@ def get_shipping_methods_for_checkout(
                     methods.extend(shipping_methods)
         return methods
 
+    def get_tax_code_from_object_meta(
```

Comment:
When we will introduce a simple taxes feature, this part should be depreceated. Wouldn't be good to create a task for that part?

Reply:
Based on the roadmap, it looks like these two features are coming in the same version. If this will work, I would consider a task that will fully remove support for tax_code in object meta in webhook plugin. As simple taxes should give an app the possibility to map the tax codes on its side.

Reply:
https://app.clickup.com/t/2549495/SALEOR-7701